docs/pkg-set.8

```diff
@@ -94,12 +94,12 @@ Also see
 .Xr pkg-updating 8
 and
 .Sx EXAMPLES .
-.It Fl x , Cm --regex
+.It Fl x , Fl -regex
 Match
 .Ar pkg-name
-as a regular expression according to the "modern" or "extended" syntax of
+as a regular expression using the modern or extended syntax of
```

Comment:
```suggestion
using an extended regular expression as documented in
```
Extended regular expression is the standardized definition of this. "Modern" is vague and doesn't age well, and this syntax has existed longer than I've been alive.